examples/graphql-context/graphql_context.bal

```diff
@@ -30,26 +29,23 @@ service /graphql on new graphql:Listener(4000) {
     function init() {
         // Initialize the `person` value.
         self.person = new("Walter White", 51, 737000.00);
-
```

Comment:
Did you check after removing these lines whether the BBE renders correctly?

Reply:
yes. it renders correctly.